File: 03-fizzbuzz/FizzBuzz.java
import java.util.ArrayList;
import java.util.List;

public class FizzBuzz {

    public static String checkFizzBuzz(int numero) {
        if (numero % 3 == 0 && numero % 5 == 0) return "FizzBuzz";
        if (numero % 3 == 0) return "Fizz";
        if (numero % 5 == 0) return "Buzz";
        return String.valueOf(numero);
    }

    public static List<String> fizzBuzz(int numero) {
        List<String> fizzBuzzList = new ArrayList<>();

        for (int i = 1; i <= numero; i++) {
            fizzBuzzList.add(checkFizzBuzz(i));
        }
        return fizzBuzzList;
    }
}

File: 03-fizzbuzz/FizzBuzz.cpp
class Solution {
public:
    vector<string> fizzBuzz(int n) {
        vector<string> result;
        bool check;
    
    for(int i = 1;i<=n;i++){
        check=false;
        result.push_back("");
        if (i%3==0)
        {
            result[i-1].append("Fizz");
            check=true;
        }
        if (i%5==0)
        {
            result[i-1].append("Buzz");
            check=true;
        }
        if (!check)
        {
            result[i-1].append(to_string(i));
        }
        
        
       
    }
     
     return result;
}
};